app/pathogen/arbovirus/visualizations/visualizations.tsx

```diff
@@ -54,7 +54,6 @@ export interface VisualizationInformation {
   urlParameter: VisualizationUrlParameter;
   displayName: VisualizationDisplayName;
   renderVisualization: () => React.ReactNode;
-  classNameWhenFullscreen: string
```

Comment:
Yeah so when I added the filter bar I took another look at the classes and figured out why this extra padding was needed and fixed it so we don't need this anymore :sparkles:

Reply:
Curious: what was the reaosn?